docs/learn-legitimate-projects.md

```diff
@@ -1,242 +1,162 @@
 ---
-id: learn-legitimate-projects
-title: How to Identify Legitimate Projects
-sidebar_label: How to Identify Legitimate Projects
+id: how-to-dyor
+title: How to Do Your Own Research (DYOR)
+sidebar_label: How to Do Your Own Research
 ---
 
-There are many projects that are being built, or intend to build, on both Polkadot and Kusama.
-However, there are also many that claim to do so, but actually have either no such intention, they
-don't have the resources to pull it through or they're outright trying to scam people by misusing
-the Polkadot and Kusama brands.
+Many projects are being built, or intend to build, on both Polkadot and Kusama. However, many claim to do so but have either no such intention, they don't have the resources to pull it through, or they're outright trying to scam people by misusing the Polkadot and Kusama brands.
 
-Telling legitimate projects from the "not so honest" ones isn't always an easy task. This guide is
-meant to help you in your own research in identifying whether a project is legitimate. What it's
-**not** meant to do, is characterise any single project as legitimate or not, or make that decision
-for you. It's simply a list of usual telltale signs of a legitimate project as opposed to one that's
-not, to help you make your own decisions.
+Telling legitimate projects from the "not so honest" ones isn't always an easy task. This guide is meant to help you find out how to do your research better when you come across a project that seems interesting. What it's **not** meant to do, is characterise any single project as legitimate or not, or make that decision for you. 
```

Comment:
Perhaps more concise: "help you in your research when you come across a project"

"label" instead of "characterise"

Reply:
> Perhaps more concise: help you in your research when you come across a project.

It's not exactly the same and I'd rather make sure it's phrased properly. It's guide to help them learn how to DYOR, not help them in their research. 

> "label" instead of "characterise"
👍